persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -179,10 +185,8 @@ public void execute() {
 
         try (Connection connection = driver.connect(databaseUrl, props)) {
             statement = connection.createStatement();
-            for (String sqlLine : sqlLines) {
-                if (!sqlLine.trim().equals("")) {
-                    statement.executeUpdate(sqlLine);
-                }
+            for (String sqlLine : sqlScript.split(";")) {
+                statement.executeUpdate(sqlLine);
```

Comment:
Yes, we can't create a table in the given order of created tables when an entity has references. We have to rearrange the existing script with a new table script. Otherwise, we will get an error when executing queries. If it is a string, we can easily re-arrange it. 
Sql script format is https://github.com/ballerina-platform/persist-tools/blob/sql-generate-script/persist-cli-tests/src/test/resources/test-src/output/tool_test_db_push_9/persist/persist_db_scripts.sql

Reply:
Where is the code that does the rearranging. IMO if it is a single string isnt it more harder to rearrange? How do you find the place to change once again? Dont you have to split the string again?

Reply:
Here, we rearrange the script. https://github.com/ballerina-platform/persist-tools/pull/60/files#diff-6c9f164aa2cc305caa4f41a77496a5e45aeb671431c49bf46b491c8c3165181bR306

Reply:
Is this a valid link?

Reply:
https://github.com/ballerina-platform/persist-tools/blob/sql-generate-script/persist-cli/src/main/java/io/ballerina/persist/utils/SqlScriptGenerationUtils.java#L306

Reply:
Updated in [48e40cb](https://github.com/ballerina-platform/persist-tools/pull/60/commits/48e40cb6c629f00934b97019c3b09567cd9e391f)